lib/components/atoms/Button/Button.style.js

```diff
@@ -19,6 +19,14 @@ const ButtonStyles = css`
     ${props.theme.border.CTA_TERTIARY};
   `
       : ``};
+  ${props =>
+    props.icon &&
+    `
+    min-height: initial;
```

Comment:
No need for **min-height: initial;** Let all button be of same height. You can overwrite this in your component as its need of your component to not have default height. Having CTA size 40x40 is a good accessibility practice. 

Secondly, If you keep top and bottom padding 0 how will icon get aligned in middle?